linkerd/docs/routers.md

```diff
@@ -0,0 +1,143 @@
+<a name="routers"></a>
+# Routers
+
+All configurations must define a **routers** key, the value of which
+must be an array of router configurations. Routers also include **servers**, which define their entry points, and **client**, which configures how clients are built.
+
+## Router Parameters
+
+<aside class="notice">
+These parameters are available to the router regardless of protocol. Routers may also have protocol-specific parameters.
+</aside>
+
+```yaml
+routers:
+- protocol: http
+  servers: ...
+  client: ...
+  announcers: ...
+  bindingCache: ...
+  label: myPackIce
+  dstPrefix: /walruses/http
+  baseDtab: |
+    /host                => /#/io.l5d.fs;
+    /walruses/http/1.1/* => /host;
+  failFast: false
+  timeoutMs: 10000
+  bindingTimeoutMs: 5000
+  responseClassifier: io.l5d.nonRetryable5XX
+```
+
+Key | Default Value | Value Description
+--- | ------------- | -----------------
+protocol | _required_ | either "[http](#http-1-1-protocol)", "[thrift](#thrift-protocol)", or "[mux](#mux-protocol-experimental)"
+servers | _required_ | a list of [server objects](#servers)
+announcers | an empty list | a list of service discovery [announcers](#announcers) that servers can announce to.
+baseDtab | an empty dtab | Sets the base delegation table. See [dtabs](https://linkerd.io/doc/dtabs/) for more.
+bindingTimeoutMs | 10 seconds | The maximum amount of time in milliseconds to spend binding a path.
+bindingCache | see [binding cache](#binding-cache) | Binding cache size configuration.
+client | an empty object | an object of [client params](#basic-client-params)
+dstPrefix | protocol dependent | A path prefix to be used on request destinations.
+failFast | `false` | If `true`, connection failures are punished more aggressively. Should not be used with small destination pools.
+interpreter | default interpreter | an [interpreter object](#interpreter) determining what module will be used to process destinations.
+label | the value of *protocol* | The name of the router (in stats and the admin ui)
+response Classifier | `io.l5d.nonRetryable5XX` | A (sometimes protocol-specific) [response classifier](#http-response-classifiers) that determines which responses should be considered failures and, of those, which should be considered [retryable](#retries).
```

Comment:
CSS question: in my browser, `io.l5d.nonRetryable5XX is currently line-wrapped as:

```
io.l5d.n
onRetrya
ble5XX
```

Would it cause problems to change the `word-wrap` or `word-break` CSS rules to avoid this?